CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The public API of this library consists of the functions declared in file
 [h3api.h](./src/h3lib/include/h3api.h).
 
 ## [Unreleased]
+### Added
+- `getBaseCells` function for getting all base cells (#174)
```

Comment:
Whoops, this should now be `getRes0Indexes` (and you might want to add a note about `res0IndexCount` as well)